src/sentry/identity/__init__.py

```diff
@@ -13,3 +13,6 @@
 exists = default_manager.exists
 register = default_manager.register
 unregister = default_manager.unregister
+
+# TODO(epurkhiser): needs to be initialized some other way
```

Comment:
does this need to be addressed before merge?

Reply:
I've updated the comment for this, but basically the idea is that this registering of the provider should happen with the slack plugin most likely.